File: java/trunk/TetramBDEJB/ejbModule/org/tetram/bdtheque/model/entity/ScenaristeAlbum.java
package org.tetram.bdtheque.model.entity;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

@Entity
@DiscriminatorValue("0")
public class ScenaristeAlbum extends AuteurAlbum {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

}

File: php/trunk/src/BDTheque/DataModelBundle/Entity/ScenaristeAlbum.php
<?php

namespace BDTheque\DataModelBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * BDTheque\DataModelBundle\Entity\ScenaristeAlbum
 *
 * @ORM\Entity
 */
class ScenaristeAlbum extends AuteurAlbum {

    public function setAlbum(Album $album) {
        $album->addScenariste($this);
        return parent::setAlbum($album);
    }

}

?>